README.md

```diff
@@ -1,3 +1,7 @@
+# Use the [lerna template](https://github.com/salesforcecli/lerna-template) if you need a library and plugin
+
+# Only use this template if you're very certain of what you are doing
```

Comment:
This seems a little heavy. What if we said `This template is useful if the library is already in another repository or the plugin is a wrapper around an API.`